CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - Remove getNavigation from Login.jsx @iRohitSingh
 - Allow listing block to be used in non-content pages (when used in a slot it
   shouldn't crash on add/edit pages) @tiberiuichim
+- [generator] Addon add prompt now accept fallacy choices @nileshgulia1
```

Comment:
Just realised this should go to the generator CHANGELOG.md

@nileshgulia1  Could you please take care?

Reply:
@sneridagh Done! Thanks!